tex2pdf-service/pyproject.toml

```diff
@@ -17,7 +17,7 @@ ruamel-yaml = "^0.18.5"
 pillow = "^10.4.0"
 python-multipart = "^0.0.6"
 psutil = "^5.9.8"
-arxiv-tex2pdf-tools = {git = "https://github.com/arXiv/submission-tools.git", subdirectory = "tex2pdf-tools" }
```

Comment:
I have a tiny concern that the branch pydantic2 will be deleted once merged. But I guess it will get the SHA in the lock file and will alert about the problem on the next update?